README.md

```diff
@@ -39,27 +39,35 @@ TODO
 
 - [tiny-invariant](https://github.com/alexreardon/tiny-invariant) - A function for assertions.
 
-### Data Structures
+### Build Tools
```

Comment:
I didn't update the Data Structures section, just moved it so that the sections are ordered alphabetically

Reply:
you'll need to update the contents too in that case FYI

Reply:
done!